src/Schema/aspire-8.0.json

```diff
@@ -73,6 +73,47 @@
               "bindMounts": {
                 "$ref": "#/definitions/bindMounts"
               },
+              "volumes": {
+                "$ref": "#/definitions/volumes"
+              },
+              "build": false
+            },
+            "additionalProperties": false
+          },
+          {
+            "type": "object",
+            "description": "A generic container resource.",
+            "required": [ "type", "image", "build" ],
+            "properties": {
+              "type": {
+                "const": "container.v1"
+              },
+              "image": {
+                "type": "string",
+                "description": "A string representing the container image to be used."
+              },
+              "entrypoint": {
+                "type": "string",
+                "description": "The entrypoint to use for the container image when executed."
+              },
+              "args": {
+                "$ref": "#/definitions/args"
+              },
```

Comment:
azd doesn't support `args` for container.v0!!  (only for project and dockerfile).  

Guessing it might not be relevant, as no one has complained so far :)

Reply:
It is, we should add it. That's a gap.

Reply:
Noted in the `container.v1` issue on the AZD side so hopefully we can get that fixed up.